### DISCUSSION TITLE
Système de marquage des joueurs inactifs

### DISCUSSION THREAD

**Discussion by Seikame** [fibreville/minotaure_php#73] [Boite à idées]
Déjà évoqué avec uurshin mais j'avais zappé de le reposter ici : 

Pour améliorer le confort de parties se déroulant sur plusieurs sessions avec le même pool de persos, pourquoi ne pas attribuer automatiquement un tag aux personnages "inactifs" à partir du moment où ils n'ont pas participés à 2 sondages ? (ou autre, mais les sondages semblent être un très bon critère)

Ce tag d'inactif disparaitrait dès la participation à un sondage. 

Le tag aurait pour principale fonction d'empêcher que le personnage inactif soit désigné par un tirage au sort pour des trucs importants (principalement les attributions des rôles : leader, traitre, etc) afin de pas bloquer certaines mécanique le temps qu'on confirme si la personne derrière est revenue ou non. 

Idéalement ce serait visible dans l'overview de l'admin, les inactifs seraient légèrement grisés par exemple, ou avec une pastille particulière.

Une autre opération potentiellement plus simple qu'un marquage automatique serait de donner à l'admin la possibilité de rajouter manuellement un paramètre pour dire "ne jamais sélectionner / désigner un inactif" : oui/non

Pourquoi pas également établir que l'attribution de ce tag supprimerait ou redistribuerait le(s) rôle(s) éventuellement détenu(s) par le personnage, dans le cas où l'inactivité se produirait pendant la session.

Enfin, j'imagine que les personnages "inactifs" ne compteraient pas dans l'affichage du pourcentage de votants.

**Comment by ElethornGM** [comment]
Super idée!

**Comment by uurshin** [comment]
Identifiée par une couleur ou une opacité différente les inactifs serait un plus oui, et faire en sorte qu'ils ne comptent plus pour toutes les opérations de désignation / nommage du coup.

Je pense qu'on peut se limiter à désigner "inactif" une personne qui ne s'est pas connecté pendant la session plutôt que se baser sur le nombre de sondage voté (plus simple et sans doute suffisant). Ca ne couvre pas les inactifs en cours de session mais c'est un cas plus mineur je pense ?

**Reply by Seikame** [reply]
Yep, si c'est plus simple à faire ça serait déjà très confortable, les autres cas seront moins fréquents surtout sur des sessions en dessous de 50 joueurs. Pour le reste ça sera juste au MJ de prendre des mesures si un des rôles est gardé trop longtemps par un inactif.
